---
title: "Labrapport"
format: html
authors: Lone, Lars, Daniel og Ulrich 
editor_options: 
  chunk_output_type: console
bibliography: references.bib
    wrap: 72
---

7 Laboratory report  

7.1 Introduction              

Analysis of gene expression through fluorescence-based real-time quantitative polymerase chain reaction (qPCR) is a well-established practice used in a variety of exercise studies (Kuang et al., 2018). This method quantifies target gene expression in biological samples, such as from blood or muscle tissue. The qPCR analysis is widely used and there are numerous protocols and methods for conducting the analysis. In our study we used the SYBR Green method where a fluorescent dye binds to the DNA during amplification. This fluorescence allows real-time tracking of the reaction, as the signal intensity correlates with the amount of DNA present (Kuang et al., 2018). The fluorescence reaches a pre-determined threshold, referred to as the cycle threshold (CT), which reflects the level of gene expression. A lower CT indicates higher gene expression, as fewer amplification cycles were required to cross the threshold (Livak & Schmittgen, 2001). 

In exercise physiology, qPCR is frequently used to investigate changes in gene expression that occur with training. In this study we aim to check primer efficiency and perform targeted amplification of cDNA using specific primers. Additionally, we aim to find out how strength training affects the gene expression of different muscle fiber types as participants underwent a two-week strength training program. The different muscle fiber types we looked at were type I, type IIa and type IIx. Type I fibers, slow-twitch, are highly oxidative and can sustain contraction over long periods without fatigue [@Pette2000]. Type IIa fibers utilize both aerobic and anaerobic metabolic pathways, have intermediate resistance to fatigue and produce more force than type I fibers [@Pette2000]. Lastly, type IIx fibers rely predominantly on anaerobic metabolism and are specialized for rapid and powerful contractions [@Pette2000].
                  

7.2 Materials  

A real-time PCR machine (We use QuantStudio 5) 

A qPCR reaction plate 

Nuclease-free water and pipette tips 

SYBR-green Master mix 

 

7.3 Method  

Prior to the experiment, the lab manager prepared the cDNA, which was extracted from samples collected during a study where participants underwent a two-week strength training program.  

First, we created a dilution series to test the primers (table 7.1). We moved 2µl from tube 1 to tube 2a, and 10µl from 1 to 2b, then vortexed tube 2a and 2b so that the sample and water (H2O) would mix. Next, we moved 2µl from 2a to 3a and 2µl from 2b to 3b, then vortexed tube 3a and 3b. Lastly, we moved 2µl from 3a to 4a and 2µ from 3b to 4b, then vortexed tube 4a and 4b. 

TABLE 7.1 

We then combined a master mix consisting of 250µl SYBR-green, 50µl primer mix (MHC1, MHC2a, MHC2x, or MHCb2m), and 100µl H2O. Subsequently, we loaded the plate with primer-specific master-mix following the outline of our pipetting scheme (table 7.2). 8µl of the master mix was added to the wells along with 2µl of cDNA sample.  

PIPETTING SCHEME – TABLE 7.2 

The plate was then covered with plastic and centrifuged at 1200rpm for 1 minute. The PCR samples were analyzed using real-time PCR (Applied Biosystems 7500 Fast Real-Time PCR System, Life Technologies AS) and QuantStudio software (Applied Biosystems, Waltham, MA, USA). The PCR process consisted of three stages: a 'Hold stage,' a 'PCR stage,' and a 'Melt curve stage.' In the first step, the temperature increased by 1.99°C/s up to 50°C, where it remained constant for 2 minutes. Then, the temperature increased by 1.99°C/s up to 95°C, where it remained constant for 2 minutes. Next, the actual PCR process began, consisting of 40 cycles. One cycle involved 1 second at 95°C, followed by the temperature dropping at a rate of 1.77°C/s to 60°C, where it remained constant for 30 seconds. After each cycle, an image of the fluorescence in the wells was taken. Finally, the temperature increased by 1.99°C/s up to 95°C and remained constant for 15 seconds. Then, the temperature dropped at a rate of 1.77°C/s to 60°C, where it remained constant for 1 minute. The temperature then increased at a rate of 0.15°C/s to 95°C, where it remained constant for 15 seconds. After the PCR process was completed, we extracted the results in the form of CT values. 

 

We analyzed our data in excel. Here we calculated the average CT value for the triplicates as well as the base-10 logarithm of the sample quantity. The slope was then calculated by plotting. Log(sample quan) against the average CT. A linear regression is performed to find the best-fit line, and the slope is derived. Primer efficiency is then calculated from the slope. Lastly, we calculated the standard deviation (SD) among replicates and the coefficient of variation (CV). 

7.4 Results  

Dilution series  

TABLE 7.3  

A slope of -2.6 indicates that the primer efficiency is not optimal as it is calculated to be 142%. We can see that our observation is not linear (Figure 7.1) 

TABLE 7.4 

The standard deviations are relatively low for the dilutions in the "pooled sample," indicating precision in the results. The standard error is lowest for the 1/2 diluted sample at 0.61% and highest for the 1/10 diluted sample at 1.58%. This suggests that there is greater variability between the measurements for the most diluted samples. Additionally, the standard error of the undiluted sample is higher than that of the 1/2 diluted sample. 

Myosin Heavy Chain 

In table 7.5 can see that the number of cycles to reach the cycle threshold (CT) has changed from week 0 to week 2. The number of cycles decreased for myhc1 from an average of 19.7 to 19.3 cycles. For myhc2a, the cycles dropped from 20.9 to 19.4, and for myhc2x it went from 26.6 to 24.4 cycles. This indicates that the gene expression increased from week 0 to week 2 as fewer cycles indicate higher gene expression (Livak and Schmittgen, 2001). 

TABLE 7.5 – CT VALUES  

In table 7.6 we can see that the amount of gene expression for the different muscle fiber types has changed from week 0 to week 2. The expression of Muscle Fiber Type 1 (myhc1) decreased from 69% to 51%, and Muscle Fiber Type 2a (myhc2a) decreased from 30% to 47%, while Muscle Fiber Type 2x (myhc2x) increased from 0.6% to 1.5%. 

TABLE 7.6: Percentage distribution of gene expression after week 0 and week 2 

 

# Diskusjon

Økningen i myhc2x, som er forbundet med de raskeste og mest eksplosive muskelfibrene, er uventet, siden denne fibertypen vanligvis reduseres ved langvarig trening til fordel for mer utholdenhets- eller styrkeorienterte fibre (type I og IIa). Det kan være at denne økningen er et resultat av kortvarige tilpasninger som gir økt eksplosivitet eller kraftutvikling, men det er også mulig at dette reflekterer tekniske feil, som for eksempel høy primer-effektivitet eller biologisk variasjon. 



# qPCR

Formålet med denne rapporten er å sjekke primereffektivitet og
gjennomføre målrettet amplifisering av cDNA ved bruk av spesifikke
primere.

## Utstyr

En sanntids PCR-maskin (vi bruker QuantStudio 5) En qPCR-reaksjonsplate
Nukleasefritt vann og pipettespisser SYBR-grønn Master mix

## Metoder

Fikk ferdig cDNA prøver fra fryser av personer som har gjennomført
systematisk styrketrening i 2 uker.

Vi fortynnet prøvene med qPCR:

```{r}
Sys.setlocale("LC_CTYPE", "en_US.UTF-8")
library(knitr)
library(kableExtra)

fortynning_tabell <- data.frame(
  Ror = c("1", "2a", "3a", "4a", "2b", "3b", "4b"),
  Fortynning = c("1", "1/10", "1/100", "1/1000", "1/2", "1/20", "1/200"),
  Prøve_volum = c("30µl", "2µl", "2µl", "2µl", "10µl", "2µl", "2µl"),
  H2O_volum = c("0µl", "18µl", "18µl", "18µl", "10µl", "18µl", "18µl")
)

kable(fortynning_tabell, caption = "Fortynningsdata", align = "c", format = "html") %>%
  kable_styling(full_width = FALSE, position = "center", bootstrap_options = c("striped", "hover", "condensed")) %>%
  column_spec(1, border_left = TRUE) %>%
  column_spec(ncol(fortynning_tabell), border_right = TRUE) %>%
  row_spec(0, extra_css = "border-bottom: 2px solid;") %>%
  kableExtra::row_spec(1:nrow(fortynning_tabell), extra_css = "border-top: 1px solid;")

```

1.  Flytter 2 µl fra rør 1 til 2a, og 10µl fra 1 til 2b, vortex rør
    2a+2b.

2.  Flytter 2µl fra 2a til 3a og 2µl fra 2b til 3b, vortex rør 3a+3b.

3.  Flytter 2µl fra 3a til 4a og 2µ fra 3b til 4b, vortex rør 4a + 4b.

Mulig avvik: første: litt lite innhold i 3b ifh til de andre

1.  Kombiner en mesterblanding:

```{r}

library(knitr)
library(kableExtra)

komponent_tabell <- data.frame(
  Komponent = c("Sybr-green 2X master-mix", "Primermix (Forward and Reverse, 5 μM each)", "H2O"),
  Volum_per_reaksjon = c("5 μl", "1 μl", "2 μl"),
  Total_volum = c("50 μl", "10 μl", "20 μl"),
  cmyc = c("125 μl", "25 μl", "50 μl")
)

kable(komponent_tabell, caption = "Komponentdata for qPCR", align = "c", format = "html") %>%
  kable_styling(full_width = FALSE, position = "center", bootstrap_options = c("striped", "hover", "condensed")) %>%
  column_spec(1, border_left = TRUE) %>%
  column_spec(ncol(komponent_tabell), border_right = TRUE) %>%
  row_spec(0, extra_css = "border-bottom: 2px solid;") %>%
  row_spec(1:nrow(komponent_tabell), extra_css = "border-top: 1px solid;")

```

-   mulig avvik i første steg: sybr-grønn: tok 200 + 50, hang igjen
    dråper i pipettespiss på begge to

-   andre steg: vond pipette for sybr-grønn 50ml

2.  Fyll platen med 8μl primerspesifikk masterblanding

3.  Tilsett 2μl cDNA-prøve.

## Pipetteringsskjema

```{r}

library(knitr)
library(kableExtra)

pipetteringsskjema <- data.frame(
  Rad = c("EN", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L"),
  Kol_13 = c("Fp1", "myhc 1", "myhc 1", "myhc 2a", "myhc 2a", "myhc 2a", "myhc 2x", "myhc 2x", "myhc 2x", "myhc", "myhc", "myhc"),
  Kol_14 = c("Fp2", "myhc 1", "myhc 1", "myhc 2a", "myhc 2a", "myhc 2a", "myhc 2x", "myhc 2x", "myhc 2x", "myhc", "myhc", "myhc"),
  Kol_15 = c("", "", "", "", "", "", "", "", "", "", "", ""),
  Kol_16 = c("", "", "", "", "", "", "", "", "", "", "", ""),
  Kol_17 = c("cmyc 1", "cmyc 1", "cmyc 1", "", "", "", "", "", "", "cmyc 1", "cmyc 1", "cmyc 1"),
  Kol_18 = c("cmyc 2a", "cmyc 2a", "cmyc 2a", "", "", "", "", "", "", "cmyc 2a", "cmyc 2a", "cmyc 2a"),
  Kol_19 = c("cmyc 3a", "cmyc 3a", "cmyc 3a", "", "", "", "", "", "", "cmyc 3a", "cmyc 3a", "cmyc 3a"),
  Kol_20 = c("cmyc 4a", "cmyc 4a", "cmyc 4a", "", "", "", "", "", "", "cmyc 4a", "cmyc 4a", "cmyc 4a"),
  Kol_21 = c("cmyc 2b", "cmyc 2b", "cmyc 2b", "", "", "", "", "", "", "cmyc 2b", "cmyc 2b", "cmyc 2b"),
  Kol_22 = c("cmyc 3b", "cmyc 3b", "cmyc 3b", "", "", "", "", "", "", "cmyc 3b", "cmyc 3b", "cmyc 3b"),
  Kol_23 = c("cmyc 4b", "cmyc 4b", "cmyc 4b", "", "", "", "", "", "", "cmyc 4b", "cmyc 4b", "cmyc 4b")
)

kable(pipetteringsskjema, caption = "Pipetteringsskjema", align = "c", format = "html") %>%
  kable_styling(full_width = FALSE, position = "center", bootstrap_options = c("striped", "hover", "condensed")) %>%
  column_spec(1, border_left = TRUE) %>%
  column_spec(ncol(pipetteringsskjema), border_right = TRUE) %>%
  row_spec(0, bold = TRUE, extra_css = "border-bottom: 2px solid;") %>%
  row_spec(1:nrow(pipetteringsskjema), extra_css = "border-top: 1px solid;")

```

MHC Delta-delta-CT method: relative quantification

```{r}

library(knitr)
library(kableExtra)

qpcr_results <- data.frame(
  Sample_name = c("FP6 R w0", "FP6 R w2pre", "FP6 R w0", "FP6 R w2pre", "FP6 R w0", "FP6 R w2pre"),
  Target_name = c("myhc1", "myhc1", "mhc2a", "mhc2a", "myhc2x", "myhc2x"),
  Ct1 = c(19.798, 18.944, 21.029, 19.549, 27.019, 24.871),
  Ct2 = c(19.901, 19.24, 21.247, 19.304, 26.898, 24.105),
  Ct3 = c(19.621, 19.861, 20.627, 19.580, 25.907, 24.256),
  Average = c(19.7733333, 19.3483333, 20.968, 19.478, 26.6079534, 24.4106166),
  Reference_gene = c("b2m", "b2m", "b2m", "b2m", "b2m", "b2m"),
  Ct1_ref = c(24.670, 22.913, 24.670, 22.913, 24.670, 22.913),
  Ct2_ref = c(24.513, 23.950, 24.513, 23.950, 24.513, 23.950),
  Ct3_ref = c(24.691, 23.819, 24.691, 23.819, 24.691, 23.819),
  Average_ref = c(24.625, 23.561, 24.625, 23.561, 24.625, 23.561),
  Delta_Ct = c(-4.851, -4.212, -3.657, -4.083, 1.983, 0.850),
  Delta_Delta_Ct = c(NA, NA, NA, NA, NA, NA),
  Two_Delta_Delta_Ct = c(28.867, 18.536, 12.613, 16.944, 0.253, 0.555)
)

kable(qpcr_results, caption = "qPCR Resultater", align = "c", format = "html") %>%
  kable_styling(full_width = FALSE, position = "center", bootstrap_options = c("striped", "hover", "condensed")) %>%
  column_spec(1, border_left = TRUE) %>%
  column_spec(ncol(qpcr_results), border_right = TRUE) %>%
  row_spec(0, bold = TRUE, extra_css = "border-bottom: 2px solid;") %>%
  row_spec(1:nrow(qpcr_results), extra_css = "border-top: 1px solid;")

```

Gene fam normalization 2\^-Ct

```{r}

library(knitr)

prosentfordeling <- data.frame(
  "100 %" = c(41.733, 36.034),
  myhc1 = c("69,17 %", "51,44 %"),
  myhc2a = c("30,22 %", "47,02 %"),
  mhc2x = c("0,61 %", "1,54 %")
)

kable(prosentfordeling, caption = "Prosentfordeling for myhc1, myhc2a og mhc2x", format = "html") %>%
  kable_styling(full_width = FALSE, position = "center", bootstrap_options = c("bordered"))

```

Etter en to uker lang styrketreningsintervensjon observeres en endring i
genuttrykket og fordeling for myhc1, myhc2a og myhc2x. Ved pre-testen
var fordelingen henholdsvis 69,17 % for myhc1, 30,22 % for myhc2a og
0,61 % for myhc2x. Etter intervensjonen, ved post-testen, endret
fordelingen seg til 51,44 % for myhc1, 47,02 % for myhc2a og 1,54 % for
myhc2x. Ct verdiene indikerer en økning av genuttrykket av myhc1, myhc2a
og myhc2x.

```{r}
library(knitr)

fortynningstabell <- data.frame(
  Fortynning = c("1", "1/2", "1/10", "1/20", "1/200"),
  Ct1 = c("28,678", "29,414", "31,776", "33,241", "Undetermined"),
  Ct2 = c("28,708", "29,620", "31,416", "32,653", "Undetermined"),
  Ct3 = c("29,155", "29,264", "32,413", "Undetermined", "34,574"),
  Avg_Ct = c("28,847", "29,433", "31,868", "32,947", "34,574"),
  Sample_quan = c(1, 0.5, 0.1, 0.05, 0.005),
  Log_sample_quan = c(0, -0.301, -1, -1.301, -2.301),
  Slope_stigningstall = c("-2,6104", "", "", "", ""),
  Primer_Efficiency = c("141,5901", "", "", "", "")
)

kable(fortynningstabell, caption = "Fortynningsrekke", format = "html") %>%
  kable_styling(full_width = FALSE, position = "center", bootstrap_options = c("bordered"))
```

Stigningstallet på -2,6 indikerer at primer effektiviteten er ikke
optimal då den er kalkulert til 142%. Vi ser at observasjonene ikke er
linjere (Figur 1)

```{r}
library(knitr)

pooled_sample <- data.frame(
  "Pooled sam" = c(1, 0.5, 0.1),
  Avg = c(28.847, 29.433, 31.868),
  SD = c(0.26704902, 0.17887641, 0.50500096),
  CV_percent = c("0,93 %", "0,61 %", "1,58 %")
)

kable(pooled_sample, caption = "Pooled Sample Data", format = "html") %>%
  kable_styling(full_width = FALSE, position = "center", bootstrap_options = c("bordered"))
```

Standardavvikene er relativt lave for fortynningene i "pooled sample"
som tyder på presisjon i resultatene. Standardfeilen er lavest for 1/2
fortynnet prøve ved 0,61% og høyest for 1/10 fortynnet prøve ved 1,58%.
Det indikerer at det er større variabilitet mellom målingene for prøvene
som er mest fortynnet. Standardfeilen til den ufortynnede prøven er
høyere enn den fortynnede 1/2.

![](images/Bilde til labrapport-01){width="472"}

#### Figur 1

## Diskusjon

Wilborn og Willoughbi (2004) sa at etter 8 uker med tung styrketrening
var genuttrykket for myhc 1 og 2a økt samt mindre for mych 2x. Det er
ikke studier på to uker med stykretrening som forklarer endringen i myhc
genuttrykk, men vi spekulerer i at det vil være samme endringer som for
Wilborn og Willoughbi (2004), men i mindre grad. Dette stemmer med våre
funn då vi ser en økning i genutrykk for både mych 1 og mych 2a, men
strider i mot at vi ser en økning i mych 2x.

Primer effektiviteten er ikke optimal då den er kalkulert til 142%.
Referanseverdiene er på 90-110% indikerer at hver PCR syklus dobbler
mengden mål DNA nøyaktig. Vår høye verdi tyder på ulike filkilder
undervegs i forsøket som feil pipettering eller kontaminasjon, primere
kan binde seg til fleire plasser på DNA og primere kan binde seg til
hverandre som kan gi ut falskt signal.

CV målingene forteller at det er høyere variabilitet i målingene når
prøven blir mer fortynnet. Gjennomsnittet er lavest på den ufortynnede
prøven noe som kan påvirke at CV er høyere enn den 1/2 fortynnede prøven
då CV reknes ut fra standardavviket delt på gjennomsnittet som gjør at
et lavere gjennomsnitt vil gi en høyere CV. Vi mistenker at dårlig
pipitering er årsak til at høyere CV for ufortynnet prøve enn 1/2
fortynnet prøve.

## Referanser

Kuang, J., Yan, X., Genders, A. J., Granata, C., & Bishop, D. J. (2018). Evaluation of iron status in non-dialysis chronic kidney disease patients. Journal of Diabetic Association Medical College, Faridpur, 7(2), 22–26. https://doi.org/10.3329/jdamc.v7i2.38245

Livak, K. J., & Schmittgen, T. D. (2001). Analysis of relative gene expression data using real-time quantitative PCR and the 2−ΔΔCT method. Methods, 25(4), 402–408. https://doi.org/10.1006/meth.2001.1262

Wilborn, C. D., & Willoughby, D. S. (2004). The role of dietary protein intake and resistance training on myosin heavy chain expression. Journal of the International Society of Sports Nutrition, 1(2), 27–34. https://doi.org/10.1186/1550-2783-1-2-27

Pette, D., & Staron, R. S. (2000). Myosin isoforms, muscle fiber types, and transitions. Microscopy Research and Technique, 50(6), 500–509. https://doi.org/10.1002/1097-0029(20000915)50:6<500::AID-JEMT7>3.0.CO;2-7
